index.html

```diff
@@ -663,6 +663,13 @@ <h2>Algorithms</h2>
 and validate status lists as described by this document.
     </p>
 
+    <p>
+If any of the algorithms in this section process a property defined in Section
+<a href="#data-model"></a> whose value is malformed, a
+<a data-cite="VC-DATA-MODEL-2.0#MALFORMED_VALUE_ERROR">MALFORMED_VALUE_ERROR</a>
+MUST be raised.
```

Comment:
I'm on the fence about this change because:

* We might want to follow the "result is a struct" pattern in the other specs and allow multiple "warnings" and "errors" to be included in the returned struct (and thus make it such that implementations don't throw errors)?
* We might want to just rely on INFRA's `#algorithm-control-flow` statement.
* Specifying this for all algorithms (across all the specs we're working on) feels a bit like repetitive boilerplate that we should avoid.

I'd appreciate strong opinions on the three options above before attempting to apply whatever is chosen uniformly across each of the specifications. These algorithms were written a LONG time ago and need a bit of a refresher to match the "new pattern", whatever that ends up becoming.

Reply:
> [@msporny] I'd appreciate strong opinions on the three options above

I don't have a strong opinion on (1) "result is a struct" vs (2) INFRA's `#algorithm-control-flow` statement.

I do think that repetitive _detailed_ boilerplate should generally be avoided, but it seems reasonable to spell things out once (somewhere) and refer to that from everywhere that might otherwise get the boilerplate. "Somewhere" might be an early/introductory REC paragraph, or a lead-in to the section where the algorithms are (if they're all together), or even an appendix ... or possibly a separate REC (since I don't think we can use a NOTE for such a thing) such that the "spell things out once" needn't happen in each-and-every document we're producing, but really does happen *once*.

Reply:
I went ahead and added that each algorithm can throw an error in 216693abb5a34997c824221e7d117b140f22b98b. 

I'll rewrite the introductory text for each algorithm in a separate PR. What we have there right now, while correct, needs significant improvement.